nekoyume/broadcast.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+import os
+from typing import Mapping, Optional
+import urllib.parse
+
+from requests import post
+from requests.exceptions import ConnectionError, Timeout
+
+from .block import Block
+from .move import Move
+from .node import Node
+from .orm import db
+
+
+DEFAULT_BROADCAST_LIMIT = os.environ.get('BROADCAST_LIMIT', 100)
+
+
+def broadcast_node(
+        serialized: Mapping[str, str],
+        sent_node: Optional[Node]=None,
+        my_node: Optional[Node]=None,
+):
+    """
+    It broadcast node to every nodes you know.
+
+    :param      serialized: serialized :class:`nekoyume.node.Node`.
+                            that will be broadcasted.
+    :param       sent_node: sent :class:`nekoyume.node.Node`.
+                            this node ignore sent node.
+    :param         my_node: my :class:`nekoyume.node.Node`.
+                            received node ignore my node when they
+                            broadcast received object.
+    """
+    for node in db.session.query(Node):
+        if sent_node and sent_node.url == node.url:
+            continue
+        try:
+            if my_node:
+                serialized['sent_node'] = my_node.url
+            url = urllib.parse.urljoin(node.url, '/nodes')
+            post(url, json=serialized, timeout=3)
+            node.last_connected_at = datetime.datetime.utcnow()
+            db.session.add(node)
+        except (ConnectionError, Timeout):
+            continue
+
+    db.session.commit()
+    return True
```

Comment:
Is this value used by somewhere else?  I guess it's not used hence okay to remove this.